interview-questions.md

```diff
@@ -7,90 +7,112 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method `super()`?
 
-  Your answer:
+  Your answer: 
+    super() is used if you're creating a new class that you want to inherit an existing class's constructors.
 
-  Researched answer:
+  Researched answer: 
+    The method super() is used when extending a parent class into a new child class you want to create. Using super() in the child class will inherit the parent class' constructors. super() can also be used to call on methods in parent classes within child classes.
 
 
 
 2. What is the DOM? What is the virtual DOM?
 
-  Your answer:
+  Your answer: 
+    DOM is Document Object Model. Not sure what virtual DOM specifically refers to.
```

Comment:
You got the right idea! Your researched answer answers the virtual DOM pretty well, id look into REACT's virtual dom and refer back to the syllabus page/recording on intro to React for some more info.